test/server.test.js

```diff
@@ -32,6 +33,9 @@ injectGlobal`
     display: flex;
     justify-content: center;
   }
+  .should-be-included {
+    content: 'css-foo matches regex and this rule is swallowed (current bug)'
+  }
 `
```

Comment:
This is a call to `injectGlobal`... not `css`. If you're viewing on GitHub, you might be having a few lines between getting clipped and that's why you're seeing it as a call to `css`

But ignore that for now - please just watch the bug in motion by changing `css-foo` in the content value to `cs-foo` and you'll now see this rule coming through in `extractCritical`.